chia/wallet/sign_coin_spends.py

```diff
@@ -46,7 +46,7 @@ async def sign_coin_spends(
 
         # Create signature
         for pk_bytes, msg in pkm_pairs_for_conditions_dict(conditions_dict, coin_spend.coin.name(), additional_data):
-            pk = blspy.G1Element.from_bytes(pk_bytes)
+            pk = blspy.G1Element.from_bytes_unchecked(pk_bytes)
```

Comment:
it looks like this is coming from a generator program. is it always  a program that our wallet just created?

Reply:
we control the private key so we know that the pk was generated properly